enhancements/single-node/single-node-openshift-day2-workers.md

```diff
@@ -0,0 +1,291 @@
+---
+title: single-node-openshift-day2-workers
```

Comment:
@omertuc I think we should structure this enhancement slightly different:
1. It should focus on adding workers as day2, and the challenges with it (user has to configure external dns/load balancer in advance
2. Then we can suggest the ingress change as an alternative
3. Also, we should add to non-goals an installation of master + worker, wdyt?
4. I think we should also cover here the infrastructure topology change that will always be single replica in such flow?

Reply:
1/2. But then that wouldn't be an enhancement, needing a load balancer to deal with the floating ingress issue is already the state of how things work currently when you add workers to a single node cluster
3. This is already the third bullet in the non-goals section
4. Nothing has to change with how the topology variables currently work if we focus on just day2. It will only be an issue if we try to deal with day-1 installation, in that case if there's more than one worker then the infra topology will change to highly available and we need to think how that's going to affect things